web/documentserver-example/java/src/main/webapp/editor.jsp

```diff
@@ -173,6 +173,27 @@
             }
         };
 
+        function onRequestRestore(event) {
+          const query = new URLSearchParams(window.location.search)
+          const config = JSON.parse('<%= FileModel.serialize(Model) %>')
```

Comment:
In this situation, I don't see a bad pattern because I'm trying to include all the necessary inside this function. I prefer not to rely on the global state of something. Ideally, the user ID, that I'm using a bit later, shouldn't be taken from the config; it should be embedded in the query.

Reply:
Fix duplication.